docs/source/extension/extension_tutorial.rst

```diff
@@ -109,7 +109,7 @@ are using to fetch pictures).
     2 - server
     3 - theme
     Choose from 1, 2, 3 [1]: 1
-    author_name [My Name]: You Name
+    author_name [My Name]: Your Name
```

Comment:
Would it makes sense to set this as a `.. code:: bash` snippet to avoid the code highlighting seen in https://jupyterlab--13593.org.readthedocs.build/en/13593/extension/extension_tutorial.html#initialize-the-project-from-a-cookiecutter

Reply:
Just fixed/pushed this.